mne/simulation/simulation_metrics.py

```diff
@@ -0,0 +1,104 @@
+# Authors: Yousra Bekhti <yousra.bekhti@gmail.com>
```

Comment:
this can just be named `metrics.py` since it's already in the `simulation` folder